programs/x509/load_roots.c

```diff
@@ -2,26 +2,14 @@
  *  Root CA reading application
  *
  *  Copyright The Mbed TLS Contributors
- *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later OR GPL-2.0-or-later
```

Comment:
There was an anomaly here: we missed this file when we moved away from GPL.
```suggestion
 *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
```

and remove the license blurbs below.